index.js

```diff
@@ -341,21 +339,27 @@ async function fastifyStatic (fastify, opts) {
       const globPattern = '**/**'
       const indexDirs = new Map()
       const routes = new Set()
-
       const winSeparatorRegex = new RegExp(`\\${path.win32.sep}`, 'g')
-
-      for (const rootPath of Array.isArray(sendOptions.root) ? sendOptions.root : [sendOptions.root]) {
+      const backslashRegex = /\\/g
+      const startForwardSlashRegex = /^\//
+      const endForwardSlashRegex = /\/$/
+      const doubleForwardSlashRegex = /\/\//g
```

Comment:
imho better style to extract them outside of all functions.

Reply:
To the top of the file?